IFC4x3/Sections/Resource definition data schemas/Schemas/IfcQuantityResource/Entities/IfcQuantityWeight/DocEntity.xml

```diff
@@ -17,7 +17,7 @@
 	</Attributes>
 	<WhereRules>
 		<DocWhereRule Name="WR21" UniqueId="eedb5c99-fdbd-458f-adb9-eac39d641365">
-			<Documentation>If a unit is given, the unit type shall be mass unit. NOTE&amp;nbsp; There is no distinction between the concept of &quot;Mass&quot; and &quot;Weight&quot; in the current IFC Release.</Documentation>
+			<Documentation>If a unit is given, the unit type shall be mass unit. &gt;NOTE&amp;nbsp; There is no distinction between the concept of &quot;Mass&quot; and &quot;Weight&quot; in the current IFC Release.</Documentation>
```

Comment:
Other documentation tags have a line break before the NOTE - is it intentional / handled differently here?